Update StarWars5E_HTML.html

Wraithmoor opened the pull request:
Added Technician Deployment

<!-- ATTENTION: This Pull Request template changed on 03/17/22. Please ensure that you are completing this template to the fullest extent possible. -->

# Submission Checklist
## Required

<!-- Check these off by adding an 'x' to each of these boxes. If you fail to meet all these criteria, your PR will be rejected. -->

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

<!-- If you are submitting a new sheet to the repository, please fill in any empty spaces indicated by < >. -->

- The name of this game is: <   >  _(i.e. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: <   > _(i.e. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: <   > _(i.e. Dungeons & Dragons, FATE)_

- [ ] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

<!-- Please check any that apply: -->

- [ ] This sheet has been made on behalf of, or by, the game's publisher.
- [ ] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: <   >
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

# Changes / Description

<!-- This is an optional step, but detailing the nature of the changes makes it easier for other contributors to track down bugs and fix issues -->

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/StarWars5E/roll20-character-sheets) *Roll20 Internal Use only.*

mperes commented:
Hey there @Wraithmoor there are some conflicts within your pull requests that need to be fixed before we can analize it. With that said, I noticed that you pretty much modified the whole content of the sheet html (17k lines), which makes think this is either a mistake or a lint issue. Could you double check those and update your PR?

Thanks for all the work you do.

nmbradley commented:
Just noting that this is still open on the basis of Miguel's comment above.

Wraithmoor commented:
I’m new to the merging process. I added one line, and it says that I changed the whole thing. 
If I just don’t have permission to edit it, I understand.
